dali/pipeline/init.cc

```diff
@@ -12,15 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <signal.h>
+#include <cstring>
 #include "dali/pipeline/init.h"
-
+#include "dali/error_handling.h"
 #include "dali/pipeline/data/backend.h"
 
 namespace dali {
 
+namespace {
+
+void signal_handler(int sig) {
```

Comment:
I don't know if this is a good idea to replace signal_handler from other aps that may use DALI.

Reply:
ok, I changed it to be enabled only if DALI_DEBUG